### ISSUE TITLE
[PROPOSAL] Sunflower Rock Functionality

### ISSUE THREAD

**Issue by WeHawt01** [sunflower-land/sunflower-land#696]
Hey! 

So this would give an interesting functionality to the Sunflower Rock. 
I will list options what the Sunflower Rock could have as functionality. 
Because it was commemorated to break Polygon why not have a functionality where :
-you would never need to sync anymore / 
-give a boost to the original sunflower plant 100% / 
-stone boost 20% more iron / 
-Increase all productivity of all ressources so plants(sunflower,potato,[...] wheat) rocks (stone, iron, gold) by 20%.

**Comment by sunfloweroglfg** [comment]
I actually like the sync concept. It would be very cool if you never has sync again or pay for syncing and it aligns w the nft! Great idea!!

**Comment by WeHawt01** [comment]
> I actually like the sync concept. It would be very cool if you never have to sync again or pay for syncing and it aligns w the nft! Great idea!!

Yeah most definitely, it has complete sence not having to sync anymore because we broke polygon and is a premium item with very low population even lower then T3 nfts. This can have a premium utility as it commemorates a previous major achievement and was in V1.

**Comment by Laoshi-Ferdy** [comment]
Another Nice Idea Here ^^

I collect it for memorize the event that we cannot forget ^^ https://twitter.com/laoshi_ferdy/status/1518224969674944515?s=20&t=fQxMYxGICv4sYM6cfL7DlQ

**Comment by BlockChainMeta** [comment]
Currently, it's not possible to never sync unless you don't want to make any progress to your farm. I do think it's a neat idea to incorporate something to this NFT, but we can brainstorm some more unique ideas. Reduction of sync fees or increased value of sunflowers or increased collection of iron may be contenders.

**Comment by RickFaker** [comment]
i think that the image nft, itself tells us that the stone was broken. what breaks stones? it's picks! i think that +20-30% when making any picks, it would be good. this is very rare nft and the bonus should be good.

**Comment by RickFaker** [comment]
Still, given the total number of sunflower stones (150 pieces), it would be wrong if the bonus would be like the beavers T1 (50,000 pieces). I think the bonus should be significant. For example, 2 times more resources when using any pickaxe.

**Comment by adamhannigan** [comment]
Thanks for the contribution and feedback! ❤️ 

 We are moving towards using an ideas portal to help capture and vote on community ideas. Github will only be used for bug/technical reports. We are only a small team and it is becoming difficult for us to respond individually to every case so we have began using community voting software.   

 If you could please upload your idea to the following community portal it would be greatly appreciated 🤗   

 https://sunflower-land.ideas.aha.io/

 You can vote on similar ideas and help bring ideas to our attention. Thanks again for being apart of this community!
